### PR TITLE
contributing document

### PR DESCRIPTION
**Pull request by tallakh**
Document describing how we want externals to contribute to this repository (issue, pr and feature requests)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+Please take a moment to review this document in order to make the contribution
+process easy and effective for everyone involved.
+
+## Using the issue tracker
+
+The issue tracker is the preferred channel for [bug reports](#bug-reports),
+[features requests](#feature-requests) and [submitting pull
+requests](#pull-requests).
+
+<a name="bugs"></a>
```
Comment (sseppola) [inline CONTRIBUTING.md:12]: What does this do?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+3. **Isolate the problem** &mdash; create a test case which is replicable.
+
+A good bug report shouldn't leave others needing to chase you up for more
+information. Please try to be as detailed as possible in your report. What is
+your environment? What steps will reproduce the issue? What browser(s) and OS
+experience the problem? What would you expect to be the outcome? All these
+details will help people to fix any potential bugs.
+
+Example:
```
Comment (sseppola) [inline CONTRIBUTING.md:35]: We could also make an issue template that pre-fills new issues with a guide like this

**Review by sseppola — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (tallakh) [reply CONTRIBUTING.md:12]: Ah, should have been removed. It was there to be able to link to a section in the file, but it works without it

File: CONTRIBUTING.md (hunk 2 above)
Comment (tallakh) [reply CONTRIBUTING.md:35]: Yes, I've created templates for feature request and bug report

**Review by henriktorget — COMMENTED** [review]
This looks good after formentioned tweaks :D

**Pull request merged**